screens/WelcomeScreen.tsx

```diff
@@ -46,7 +46,8 @@ function WelcomeScreen({ navigation }) {
                     </View>
                 </LinearGradient>
             </Pressable>
-            <Pressable style={styles.demoContainer}>
+            <View style={styles.demoShadowContainer} />
```

Comment:
There is no corresponding style for this View + `demoContainer` defines its own shadow. Was there another purpose in mind for this empty View?